elements/lisk-api-client/src/transaction.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import {
+	signTransaction,
+	signMultiSignatureTransaction,
+	computeMinFee,
+} from '@liskhq/lisk-transactions';
+import { getAddressAndPublicKeyFromPassphrase } from '@liskhq/lisk-cryptography';
+import {
+	decodeTransaction,
+	encodeTransaction,
+	getTransactionAssetSchema,
+	decodeAccount,
+} from './codec';
+import { Channel, RegisteredSchemas, NodeInfo } from './types';
+
+export interface CreateTransactionOptions {
```

Comment:
if we export it then we can move it to `types.ts`?

Reply:
but it's only being used in `transaction.ts` so it shouldn't go to `types.ts` imo

Reply:
we are keeping all the exported types in `types.ts`, if not used elsewhere we can simply remove export keyword